shell/common/api/electron_api_native_image.cc

```diff
@@ -122,6 +122,10 @@ base::win::ScopedGDIObject<HICON> ReadICOFromPath(int size,
 }
 #endif
 
+[[nodiscard]] v8::Local<v8::Object> NewEmptyBuffer(v8::Isolate* isolate) {
```

Comment:
Return type could be just `v8::Local<v8::Value>`, any reasons to specialize it ?

Reply:
Good call. Fixed in 5697e70f800499adfce64cd29358fc470f29bde0